clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/AbstractStoreBufferService.java

```diff
@@ -2,16 +2,17 @@
 
 import com.linkedin.venice.kafka.protocol.KafkaMessageEnvelope;
 import com.linkedin.venice.message.KafkaKey;
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import com.linkedin.venice.pubsub.api.PubSubTopicPartition;
 import com.linkedin.venice.service.AbstractVeniceService;
-import org.apache.kafka.clients.consumer.ConsumerRecord;
 
 
 /**
  * Abstract class capturing the responsibilities of drainers threads doing store ingestion.
  */
 public abstract class AbstractStoreBufferService extends AbstractVeniceService {
   public abstract void putConsumerRecord(
-      ConsumerRecord<KafkaKey, KafkaMessageEnvelope> consumerRecord,
+      PubSubMessage<KafkaKey, KafkaMessageEnvelope, Long> consumerRecord,
```

Comment:
Not a real comment, but wondering if we should change the variables and function names to match the new type naming?  I don't feel strongly about it.

Reply:
Are you sure :D ...?

```
grepcode 'consumerRecord' | wc -l
     327
```

Reply:
LMK how you'd like it to be renamed and I can give it a go.

Reply:
OffTopic, but I'm also interested to know what `grepcode` is an alias for

Reply:
Down this way lies madness.  We can go a bit at a time across other diffs lol